pkg/kapp/resources/resources.go

```diff
@@ -564,6 +564,16 @@ func IsResourceChangeBlockedErr(err error) bool {
 	}
 }
 
+func IsPlaceholderResourceExistenceErr(err error) bool {
+	errMsg := err.Error()
+	switch {
+	case strings.Contains(errMsg, "Placeholder resource doesn't exists"):
```

Comment:
since this is used twice. is there a case to extract this and make an error's constant and use the constant here?

Reply:
I just thought that this error won't be used anywhere else apart from these 2 scenarios, so didn't use a constant. But I guess there's no harm in doing that. Will make the change.

Reply:
I would suggest something like `IsPlaceholderError()` in the same file as the error itself. And maybe that can be consumed by this function? Similar to how api-machinery exposes functions like `IsStatusInternalError()`

Reply:
I had thought of creating a function called `IsPlaceolderError()` and consuming it directly while checking for retries. 
Currently we are importing `resources` package inside `clusterapply`, so we won't be able to import `clusterapply` inside `resources`.

Reply:
Ah a circular dependency. Did not think of that. Makes sense